### ISSUE TITLE
[Bug]: Audio is REALLY quiet

### ISSUE THREAD

**Issue by HairyMilkshakes** [FreeTubeApp/FreeTube#5644]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Watch this video in browser https://youtu.be/JQqGGgEJHqk
2. It's loud
3. Now watch this video in Freetube program
4. It's 1/4 as loud
5. Volume is turned all the way up
![GBGlKUV1cW](https://github.com/user-attachments/assets/315acde0-d744-4197-86ad-67a77568bf85)
Using this build: https://github.com/FreeTubeApp/FreeTube/actions/runs/10688288090

### Expected Behavior

Same volume from browser and in program.

### Issue Labels

accessibility issue

### FreeTube Version

freetube-0.21.3-nightly-4711

### Operating System Version

W10

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

From maybe 3 days ago (build tab)

### Additional Information

None

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
Click on the audio track selector and selector and select Stable Volume.

**Comment by HairyMilkshakes** [comment]
> Click on the audio track selector and selector and select Stable Volume.

One more question before you close it again - the audio track defaults to original and not stable volume. Any way for us to set this in settings? Don't see it so far, thanks.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Not at this time see https://github.com/FreeTubeApp/FreeTube/issues/4291

**Comment by absidue** [comment]
@ngocphamm It's a YouTube feature, so I suggest looking it up with your search engine :)

**Comment by ngocphamm** [comment]
@absidue I didn't know it's a Youtube feature and should have tried searching first. Sorry.
